src/device/execution.jl

```diff
@@ -1,110 +1,8 @@
 # kernel execution
 
-export AbstractGPUBackend, AbstractKernelContext, gpu_call
-
-abstract type AbstractGPUBackend end
-
-abstract type AbstractKernelContext end
```

Comment:
Is the intention to remove `AbstractKernelContext` completely?

Reply:
Yeah I think so. It shouldn't be needed anymore.